winml/lib/Api/LearningModel.cpp

```diff
@@ -279,6 +279,13 @@ LearningModel::OutputFeatures() try {
 }
 WINML_CATCH_ALL
 
+void LearningModel::SetName(const hstring& name) try {
+  auto name_std_str = _winml::Strings::UTF8FromHString(name);
```

Comment:
what does this return when you pass in something that can't be represented in char*? like a wide-unicode character